applications/serial_lte_modem/src/dfu/dfu_host.c

```diff
@@ -60,13 +60,15 @@ int dfu_host_send_fw(const uint8_t *data, uint32_t data_size)
 			break;
 		}
 
+#if defined(CONFIG_SLM_LOG_LEVEL_DBG)
 		if (crc_32 == 0) {
```

Comment:
crc_32 should be declared under _LOG_LEVEL_DBG, too.

Reply:
declared crc_32 inside _LOG_LEVEL_DBG.